docs/app/Views/Usage.js

```diff
@@ -0,0 +1,116 @@
+      </Header>
+    </Segment>
+
+    <Segment basic padded>
+      <Header as='h2' dividing>Usage</Header>
+      <p>
+        The Semantic UI React package can be installed via NPM:
+      </p>
+      <Segment>
+        <pre>$ npm install {pkg.name}</pre>
+      </Segment>
+      <p>
+        Installing Semantic UI React provides the JavaScript for your components.
+        You'll also need to include a stylesheet to provide the styling for your components.
+        This is the typical pattern for component frameworks, such as Semantic UI or Bootstrap.
+      </p>
+      <p>
+        The method you choose to include the stylesheet in your project will depend on the level
+        of customisation you require.
+      </p>
+    </Segment>
+
+    {/* ----------------------------------------
+     *  Content Delivery Network (CDN)
+     * -------------------------------------- */}
+    <Segment basic padded>
+      <Header as='h2' dividing>Content Delivery Network (CDN)</Header>
+      <p>
+        You can use the default Semantic UI stylesheet by including a Semantic UI CDN link in your
+        <em>index.html</em> file.
+      </p>
+      <p>
+        This is the quickest way to get started with Semantic UI React. You won't be able to use
+        custom themes with this method.
+      </p>
+      <Segment>
+        <pre>
+          {'<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.4/semantic.min.css"></link>'}
+        </pre>
+      </Segment>
+    </Segment>
+
+    {/* ----------------------------------------
+     *  Semantic-UI-CSS package
+     * -------------------------------------- */}
+    <Segment basic padded>
+      <Header as='h2' dividing>Semantic UI CSS package</Header>
+      <p>
+        The <a href={semanticUICSSRepoURL}>Semantic UI CSS package</a> is automatically synced with the
+        main Semantic UI repository to provide a lightweight CSS only version of Semantic UI.
+      </p>
+      <p>
+        Semantic UI CSS can be installed as a package in your project using NPM. You won't be able to
+        use custom themes with this method.
+      </p>
+      <Segment>
+        <pre>
+          $ npm install semantic-ui-css
+        </pre>
+      </Segment>
+    </Segment>
+
+    {/* ----------------------------------------
+     *  Semantic-UI package
+     * -------------------------------------- */}
+    <Segment basic padded>
+      <Header as='h2' dividing>Semantic UI package</Header>
+      <p>
+        Install the full <a href={semanticUIRepoURL}>Semantic UI package</a>.
+      </p>
+      <p>
+        Semantic UI includes Gulp build tools so your project can preserve its own theme changes,
+        allowing you to customise the style variables.
+      </p>
+      <p>
+        Detailed documentation on theming in Semantic UI is provided
+        <a href={`${semanticUIDocsURL}usage/theming.html`}>here</a>.
+      </p>
+      <Segment>
+        <pre>
+          $ npm install semantic-ui
```

Comment:
This one should probably be `--save-dev` given it installs a build system and the final stylesheet is a product of that build.